ibis/pandas/context.py

```diff
@@ -0,0 +1,97 @@
+import abc
+import operator
+
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class Context(object):
```

Comment:
Gotcha. The name doesn't matter too much for now as long as it's explained what the thing is.